apps/wallet/src/ui/app/pages/approval-request/transaction-request/DescriptionList.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * TODO: Generalize this file into shared components.
+ */
+
+import type { ReactNode } from 'react';
+
+export interface DescriptionItemProps {
```

Comment:
Yes but currently we have no consistency here and we'd need to agree which one first.

It's exported because it was copied from the UI components from explorer. I can un-export but then I'll re-export it when we generalize it so seems fine either way.